src/access-controllers.js

```diff
@@ -3,10 +3,11 @@
 const AccessControllerManifest = require('./access-controller-manifest')
 const IPFSAccessController = require('./ipfs-access-controller')
 const OrbitDBAccessController = require('./orbitdb-access-controller')
+const ContractAccessController = require('./contract-access-controller')
```

Comment:
We don't need to add the module here, it's supposed to be required by the user and pass in through addAccessController.